pymysqlreplication/event.py

```diff
@@ -477,6 +557,14 @@ def _dump(self):
         print("Execution time: %d" % (self.execution_time))
         print("Query: %s" % (self.query))
 
+    def _to_json(self):
+        return {
+            "event_name": type(self).__name__,
+            "schema": self.schema.decode("utf-8"),
```

Comment:
I think I think it's better to send it in the same form as the dump, but what do you think?